packages/documents/graphql/schema-types.js

```diff
@@ -50,9 +50,23 @@ type Meta {
   facebookTitle: String
   facebookImage: String
   facebookDescription: String
+  facebookGenerated: Boolean
+  facebookText: String
```

Comment:
(optional) Could we combine this in more prop? And maybe `facebookImageText` to be a bit more distinct?

Reply:
Yeah... that's more of a convenience/clarity prop. Otherwise I have to manually erase any text when the producer switch to a custom image upload. Might also be useful to have those 2 decoupled if we want to derive a default text from something else (e.g. title, use twitter text for facebook, etc.)